extensions/adobe/experience/decisioning/activity-detail.schema.json

```diff
@@ -7,41 +7,46 @@
   ],
   "$id": "https://ns.adobe.com/experience/decisioning/activity-detail",
   "$schema": "http://json-schema.org/draft-06/schema#",
-  "title": "Decision Activity Detail",
+  "title": "Decision Activity Snapshot",
   "type": "object",
   "meta:extensible": true,
   "meta:abstract": true,
-  "description": "Detail about the entity defining and controlling the decisioning process. Decisions are made in the context of an activity. At any given time, the best option is re-evaluated, or ranked, based on the most current set of context variables, rules and constraints. The decision activity specifies a fallback option should the combined constraints disqualify all narrowed-down options.",
+  "description": "Snapshot of the properties of the decision activity. A decision activity is an entity that controls a decisioning process. Decisions are made in the context of an activity. See definition of decision activty for more details.",
   "definitions": {
     "activity-detail-datatype": {
       "properties": {
         "xdm:id": {
           "title": "Activity Identifier",
           "type": "string",
           "format": "uri-reference",
-          "description": "A unique identifier for the decision activity."
+          "description": "A unique, immutable identifier for the decision activity."
+        },
+        "xdm:version": {
```

Comment:
@prabhum2  and @khanna-vikas  - Is this to become the official definition of xdm:version field? Or should this be named xdm:activityVersion or something since it's related to the activity?

Reply:
I moved to a different property name "repo:etag". It is used elsewhere and is used in the same way we are using it here: to denote a particular snapshot in time for an entity/resource